From 432c281b43c09c39ed1b9dc0302bc4cdbc653f1b Mon Sep 17 00:00:00 2001
From: Kyle Harding <kyle@balena.io>
Date: Tue, 9 Mar 2021 10:23:23 -0500
Subject: [PATCH 2/2] Rename flasher flag path to balena-image-flasher

As part of a full rename away from legacy resin namespaces the
following os-config compatibility changes are required to align
with meta-balena changes.

Signed-off-by: Kyle Harding <kyle@balena.io>
---
 src/args.rs          | 2 +-
 tests/integration.rs | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/args.rs b/src/args.rs
index cea1647..2ef8c28 100644
--- a/src/args.rs
+++ b/src/args.rs
@@ -11,7 +11,7 @@ const CONFIG_ROUTE: &str = "/os/v1/config";
 const OS_CONFIG_PATH: &str = "/etc/os-config.json";
 const CONFIG_JSON_PATH: &str = "/mnt/boot/config.json";
 const CONFIG_JSON_FLASHER_PATH: &str = "/mnt/boottmp/config.json";
-const FLASHER_FLAG_PATH: &str = "/mnt/boot/resin-image-flasher";
+const FLASHER_FLAG_PATH: &str = "/mnt/boot/balena-image-flasher";
 
 const CONFIG_ROUTE_REDEFINE: &str = "CONFIG_ROUTE_REDEFINE";
 const OS_CONFIG_PATH_REDEFINE: &str = "OS_CONFIG_PATH_REDEFINE";
diff --git a/tests/integration.rs b/tests/integration.rs
index c1c7a87..9faf6a1 100644
--- a/tests/integration.rs
+++ b/tests/integration.rs
@@ -424,7 +424,7 @@ fn join_flasher() {
 
     let service_1 = MockService::new(unit_name(1), &script_path);
 
-    let flasher_flag_path = create_tmp_file(&tmp_dir, "resin-image-flasher", "", None);
+    let flasher_flag_path = create_tmp_file(&tmp_dir, "balena-image-flasher", "", None);
 
     let config_json = r#"
         {
-- 
2.27.0

